### ISSUE TITLE
Unexpected Behavior based on Readme

### ISSUE THREAD

**Issue by DeepFriedDerp** [EngineersBox/MC-BE-LevelDB-Key-Calculator#1]
Situation:
- Lenovo ThinkPad running Windows 10
- Git cloned onto local, in main branch
- Installed Go and restarted both command prompt and git bash terminals
- cd into cloned repo
- attempted example run command, go performs initial dependency download/install, returns "exit status 1"
- attempted command again, exit status 1
- attempted similar command, leaving a lot of options to their default state, exit status 1
- attempted similar set of commands on git-bash, each time returns exit status 1

Let me know if you need any additional info.

**Comment by EngineersBox** [comment]
Hiya,

Thanks for the issue. Turns out it was just case sensitivity on the `-type` flag that was unecessary. I've updated the repo to ignore the case of that flag. Note that the `-tag` flag still requires case sensitivity.

I can run the examples now without issue using any case:
```shell
go run main.go hexkey -x 413 -z 54 -y 104 -type nEtHeR -tag PendingTicks
1900000030000000ffffffff3306
```

Please pull the latest commits, and it should work for you now.

Cheers,
EngineersBox

**Comment by EngineersBox** [comment]
Fixed in release [v2.0.1](https://github.com/EngineersBox/MC-BE-LevelDB-Key-Calculator/releases/tag/v2.0.1)
